core/node/events/miniblock_info.go

```diff
@@ -542,3 +542,24 @@ func (p *ParsedMiniblockInfoOpts) WithApplyOnlyMatchingSnapshot() *ParsedMiniblo
 func (p *ParsedMiniblockInfoOpts) ApplyOnlyMatchingSnapshot() bool {
 	return p.applyOnlyMatchingSnapshot
 }
+
+func MiniblockInfosToStorageMbs(mbs []*MiniblockInfo) ([]*storage.MiniblockDescriptor, error) {
+	storageMbs := make([]*storage.MiniblockDescriptor, len(mbs))
+	var err error
+	for i, mb := range mbs {
+		if i > 0 {
+			if mb.Ref.Num != mbs[i].Ref.Num+int64(i) ||
```

Comment:
`mb.Ref.Num != mbs[i].Ref.Num+int64(i)`
Should that not be?
`mb.Ref.Num != mbs[0].Ref.Num+int64(i)`